dfs/nfs/mounts_test.go

```diff
@@ -11,7 +11,7 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-// +build unit
+// +build unit, linux
```

Comment:
I don't have any OSX support for several things like NFS and devicemapper.  So if I follow the `_platform`  naming convention then we end up with many of the files in that package name `someName_linux.go` as compared to the `utils` package where the majority of the business logic is platform neutral